src/containers/views/Documents.js

```diff
@@ -98,12 +98,19 @@ export class Documents extends Component {
       return null;
     }
 
+    let newBtn = null;
+    if (collection_name == "posts") {
+      newBtn = "New Post";
+    } else {
+      newBtn = "New Document";
+    }
+
     return (
       <div>
         <div className="content-header">
           <h1>{capitalize(collection_name)}</h1>
           <div className="page-buttons">
-            <Link className="btn btn-active" to={`${ADMIN_PREFIX}/collections/${collection_name}/new`}>New post</Link>
+            <Link className="btn btn-active" to={`${ADMIN_PREFIX}/collections/${collection_name}/new`}>{newBtn}</Link>
```

Comment:
Feel free to commit to my branch. I need more experience points to modify tests.. :grin: